schema/schema.yaml

```
@@ -1,48 +1,37 @@
 $schema: https://fiboa.github.io/schema/v0.1.0/schema.json
 required:
```

```suggestion
required:
  - determination_method
  - determination_datetime
```

Do you also want to require area and perimeter or are these optional for AI use cases?

---

Ah that's how you do it – simpler than I imagined, haha. I would say it doesn't hurt to have area and perimeter, but I think we can leave it optional for now – it really depends on if we incorporate metadata when training or not.